values.go

```diff
@@ -38,6 +39,12 @@ func GetFlags(dir string) (map[string]string, error) {
 
 	if version, err := versionFromFile(dir); err != nil {
 		return nil, fmt.Errorf("failed to get version: %v", err)
+	} else if version == "" {
+		// ex: v0.1.2 0.1.2 0.1.2.dev3
+		matched, err := regexp.MatchString(`^v?(\d+\.){1,3}(\d+)`, gitSummary)
```

Comment:
> v1.0-rc1 will still match to v1.0-rc1 not v1.0. Are you missing understanding?

ok I just realized you used Match*; not Find*. I don't think we can help people misusing tagging. So I'm in favor of removing the regexp. Just providing `repo.Summary()` will give users an irrelevant tag; but then they should not use bad tags. We can't fix that by not giving a Version string.

Let's just call `repo.Summary()`.